javascriptv3/example_code/bedrock-runtime/models/ai21_labs_jurassic2/jurassic2.js

```diff
@@ -0,0 +1,78 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import { fileURLToPath } from "url";
+
+import { FoundationModels } from "../../config/foundation_models.js";
+import { defaultProvider } from "@aws-sdk/credential-provider-node";
+import {
+  BedrockRuntimeClient,
+  InvokeModelCommand,
+} from "@aws-sdk/client-bedrock-runtime";
+
+/**
+ * @typedef {Object} Data
+ * @property {string} text
+ *
+ * @typedef {Object} Completion
+ * @property {Data} data
+ *
+ * @typedef {Object} ResponseBody
+ * @property {Completion[]} completions
+ */
+
+/**
+ * Invokes an AI21 Labs Jurassic-2 model.
+ *
+ * @param {string} prompt - The input text prompt for the model to complete.
+ * @param {string} [modelId] - The ID of the model to use. Defaults to "ai21.j2-mid-v1".
+ * @returns {Promise<string[]>} The inference response from the model.
+ */
+export const invokeModel = async (prompt, modelId) => {
+  // Create a new Bedrock Runtime client instance.
+  const client = new BedrockRuntimeClient({
+    region: "us-east-1",
+    credentialDefaultProvider: defaultProvider,
+  });
+
+  // Use the provided model ID or fallback to Jurassic-2 Mid if not provided.
+  modelId = modelId || "ai21.j2-mid-v1";
```

Comment:
This can be handled in the function sig.
```
export const invokeModel = async (prompt, modelId = "ai21.j2-mid-v1") => {
```

That said, should we even allow this as a parameter? Someone could easily put in a model id that won't work with this usage.

Reply:
- Re Model ID as paramter: The tests inject different model ids into the same invokers (e.g. Claude 3 Haiku and Claude 3 Sonnet, or Llama2 13B and Llama2 70B). It's true that they fail if the wrong model ID is used. 
- Re default in signature vs function body: I figured it's more visible in a piece of example code this way but I can add is as default in the sig

Reply:
Might be more visible but will feel awkward to most JS devs I would think.